docs/v1/src/pages/docs/why.en.mdx

```diff
@@ -9,7 +9,7 @@ import { Steps } from 'nextra/components'
 If you have used React Suspense in SSR environment like Next.js, you may encounter error like below.
 ![Example banner](/img/suspense-in-ssr-error.png)
 
-It's why [`<Suspense/>` and `<Suspense.CSROnly/>`](/docs/react/Suspense) is added in this library.
+It's why [`<Suspense csrOnly/>`](/docs/react/Suspense) is added in this library.
```

Comment:
What about this?

```suggestion
In Suspensive, you can work around this with the `csrOnly` prop.
```

in kr

```md
Suspensive에서는 이와 같은 문제를 csrOnly prop을 통해 해결할 수 있습니다.
```

Reply:
For keep consistency, I didn't update it as you comment.